README.rst

```diff
@@ -875,19 +875,62 @@ Get command details
 
     GET /api/v1/controller/device/{device_id}/command/{command_id}/
 
-Get device coordinates
+Get device location
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    GET /api/v1/controller/device/{id}/location/?key={device key}
+
+**Note**: Device Location endpoints can only be accessed with the
+Device ``key`` or User ``Token``. When there is no location associated
+with the device with ``GET`` request a location of ``outdoor`` type
+will get automatically created with null geometry value, and then the
+data of the locaiton, floorplan and indoor coordinates can be manipulated.
```

Comment:
These endpoints should be accessible in 2 ways (either one or the other):

- as all the other endpoints (token or session), when this happens permissions and multi-tenancy shall be respected
- device key, we can assume the device itself it's updating its position so we do not have to check multi-tenancy or permissions

Please document it as described above, I also suggest to make a subheading for this explanation and link to it from all the device location endpoints.

Reply:
Here, One doubt, what if a user is authenticated, and provides a correct ID and key of the device belonging to different organization, We should still respect multitenancy right???

Reply:
Done, Have added it